### ISSUE TITLE
Template does not knit

### ISSUE THREAD

**Issue by bttomio** [ulyngs/oxforddown#19]
Hi,

While running the template, I am getting this error message:

```
Quitting from lines 759-762 (_main.Rmd) 
Error in 1:header_rows_end : NA/NaN argument
Calls: <Anonymous> ... kable_styling -> pdfTable_styling -> styling_latex_repeat_header
In addition: Warning messages:
1: In yaml::yaml.load(..., eval.expr = TRUE) :
  NAs introduced by coercion: 10,052 is not an integer
2: In yaml::yaml.load(..., eval.expr = TRUE) :
  NAs introduced by coercion: 10,052 is not an integer
Execution halted
```

Could you please check if I am doing anything wrong? Thanks a lot!

Here is my session information:

```
R version 4.0.5 (2021-03-31)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.2 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=fr_FR.UTF-8        LC_COLLATE=en_US.UTF-8     LC_MONETARY=fr_FR.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=fr_FR.UTF-8       LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=fr_FR.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] compiler_4.0.5    bookdown_0.21     htmltools_0.5.1.1 tools_4.0.5       yaml_2.2.1        rmarkdown_2.7     knitr_1.32        xfun_0.22         digest_0.6.27    
[10] rlang_0.4.10      evaluate_0.14    
```

**Comment by bttomio** [comment]
It's related to knitr_1.32. It works fine after reversing the update, i.e. using knitr_1.31. Thanks.

**Comment by bttomio** [comment]
Solved with knitr 1.3.5: https://github.com/yihui/knitr/issues/1595#issuecomment-823753382. Thanks, @yihui!
